Dockerfile

```diff
@@ -3,4 +3,12 @@ COPY ./requirements.txt /dockerBuild/requirements.txt
 WORKDIR /dockerBuild
 RUN pip install --no-cache-dir -r requirements.txt
 COPY . /dockerBuild
+ENV DB_TYPE=postgresql
```

Comment:
Don't we need to use the same docker env for db postgres that being used in BIT backend?
<img width="392" alt="Screen Shot 2021-03-10 at 9 56 02 pm" src="https://user-images.githubusercontent.com/29667122/110619167-96aab680-81eb-11eb-8206-58b2a5054cff.png">

Bcoz that's how the 2 backends connect to the postgres locally.

Reply:
Yes it is the same host. Let me see where the POSTGRES_HOST and the POSTGRES_PORT and being used and i'll get back to you , but what I can tell you is that they are connected to the same postgre database.